src/ejabberd_sm.erl

```diff
@@ -1074,9 +1068,4 @@ get_cached_unique_count() ->
 
 -spec sm_backend() -> backend().
 sm_backend() ->
```

Comment:
With the new changes, is this function even used? If it's not, I don't see a point keeping it, as HostTypes (or, until it's changed, `global`) should be enough to determine which backend is used.

Reply:
it is used in tests